api/v1/trustypb/cis.proto

```diff
@@ -29,8 +29,6 @@ enum Trust {
 message RootCertificate  {
     // ID of the certificate
     int64 id = 1 [(gogoproto.customname)="ID"];
-    // OrgID of the certificate, only used with Org scope
-    int64 org_id = 2 [(gogoproto.customname)="OrgID"];
     // Skid provides Subject Key Identifier
     string skid = 3 [(gogoproto.customname)="SKID"];
```

Comment:
is it ok that now we jump from 1 to 3? I guess that is because you cannot change IDs once they are defined?

Reply:
fixed